Mathlib/LinearAlgebra/Finsupp/LinearCombination.lean

```diff
@@ -76,6 +76,13 @@ variable (α M)
 theorem linearCombination_zero : linearCombination R (0 : α → M) = 0 :=
   LinearMap.ext (linearCombination_zero_apply R)
 
+@[simp]
+theorem linearCombination_single_index (c : M) (a : α) (f : α →₀ R) :
+    linearCombination R (single a c) f = f a • c := by
```

Comment:
To reiterate my previous comment, the second argument to `linearCombination` is a function not a finsupp, so this is a more natural spelling
```suggestion
    linearCombination R (Pi.single a c) f = f a • c := by
```

(Finsupp.single_eq_pi_single converts between the spellings)

Reply:
I don't understand, Pi.single gives an error, a `DecidableEq alpha` is missing…

Reply:
I added that, anyway.